bats/tests/k8s/helm-install-rancher.bats

```diff
@@ -98,22 +163,65 @@ deploy_rancher() {
     helm upgrade \
         --install cert-manager jetstack/cert-manager \
         --namespace cert-manager \
-        --set installCRDs=true \
+        --set crds.enabled=true \
+        --set crds.keep=true \
+        --set prometheus.enabled=false \
         --set "extraArgs[0]=--enable-certificate-owner-ref=true" \
         --create-namespace
+    try assert_not_empty_list helm list --namespace cert-manager --deployed --output json --selector name=cert-manager
+    wait_for_kube_deployment_available --namespace cert-manager cert-manager
 
     local host
     host=$(traefik_hostname) || return
```

Comment:
Given that we don't add `|| return` on the other statements that can fail here (like the `try` and `wait_for*` lines just above, we don't need `|| return` here either.

I don't really care too much, but notice the inconsistency.